addons/website_profile/controllers/main.py

```diff
@@ -38,7 +38,7 @@ def _check_avatar_access(self, user_id, **post):
     def _get_default_avatar(self):
         img_path = modules.get_module_resource('web', 'static/img', 'placeholder.png')
         with open(img_path, 'rb') as f:
```

Comment:
yes please, this can be simplified and corrected as:

```py
    def _get_default_avatar(self):
        with tools.file_open("web/static/img/placeholder.png", 'rb') as f:
            return f.read()
```